cmd/vcluster/cmd/start.go

```diff
@@ -85,6 +91,21 @@ func ExecuteStart(ctx context.Context, options *StartOptions) error {
 		return fmt.Errorf("license init: %w", err)
 	}
 
+	// check if there are existing vclusters in the current namespace. This can be achieved by listing the vcluster services
+	vclusterServices, err := listVclusterServices(ctx, vConfig.ControlPlaneClient, vConfig.ControlPlaneNamespace)
+	if err != nil {
+		return err
+	}
+
+	// add a deprecation warning if reuseNamespace flag is being set to true
+	if !vConfig.ReuseNamespace {
+		if len(vclusterServices) > 0 {
+			return fmt.Errorf("there is already a virtual cluster in the current namespace %s", vConfig.ControlPlaneNamespace)
```

Comment:
This shouldn't error out yet, just log a warning. We'll start erroring in 0.24. I believe this is why all the tests are failing.

Reply:
I've made the above change.
But just for my understanding, so this means that there'll be a difference in vcluster creation behaviour via helm vs. via CLI. 
For helm if the user creates another vcluster **even without using the reuse-ns config** i.e. `reuse-ns=false`, we'll allow it with a warning. 
But for CLI, if the user does the same then we'll error out because for CLI this is not a new flag or feature. Am I correct?